### ISSUE TITLE
Language keys not compiling

### ISSUE THREAD

**Issue by bryceosterhaus** [brunobasto/liferay-watch#7]
Seems that after I run lwatch, all of our language keys are no longer translated.

**Comment by brunobasto** [comment]
I might know why. In portal's build system, we have a task called ``replaceSoyTranslation`` that replaces calls to Google's default translation mechanisms and uses our ``Liferay.Language.get`` method instead. We need to either call gradle (which might slow things down) or implement it ourselves.

**Comment by bryceosterhaus** [comment]
Yeah I figured it might cost us some speed. Its not a huge deal for us though, just wanted to make an issue just in case.
